.github/workflows/docker.yaml

```diff
@@ -26,34 +39,49 @@ jobs:
               -e DREMIOAI_DREMIO__OAUTH_SUPPORTED=false \
           dremio-mcp-server:${{ github.sha }} \
             dremio-mcp-server tools list
-      - name: Install uv
-        run: |
-           curl -LsSf https://astral.sh/uv/install.sh | sh
-           echo "$HOME/.cargo/bin" >> $GITHUB_PATH
-
-      - name: Set up Python
-        uses: actions/setup-python@v4
-        with:
-          python-version: '3.13'
-
       - name: Start container
         run: |
           docker run \
+            --detach \
             -e DREMIOAI_TOOLS__SERVER_MODE=FOR_DATA_PATTERNS \
             -e DREMIOAI_DREMIO__URI=https://fake \
             -e DREMIOAI_DREMIO__OAUTH_SUPPORTED=false \
             -p 6789:6789 \
-            --name mcp --rm \
+            --name mcp \
+            --rm \
             --network host \
           dremio-mcp-server:${{ github.sha }} \
           dremio-mcp-server run --enable-streaming-http --port 6789 --no-log-to-file \
-            --enable-json-logging &
+            --enable-json-logging
 
       - name: Test container
         run: |
           uv run python tests/stremable_http_cli.py --url http://127.0.0.1:6789/mcp --token fake
+      - id: tag
+        name: Create tag
+        run: |
+          git config --global --add safe.directory /__w/dremio-mcp/dremio-mcp
+
+          head_sha=$(git rev-parse --short HEAD)
+          timestamp=$(date -u +"%Y%m%d%H%M%S")
 
-      - name: Stop container
+          # This tag follows the same convention as https://github.com/dremio/opensearch-k8s-operator/blob/main/.github/workflows/publish.yaml#L17
```

Comment:
@eduardo-munoz-dremio - dremoio-mcp is a OSS repo. Should we remove this comment referring to non-OSSz